video/profiles.go

```diff
@@ -93,11 +93,11 @@ func GetPlaybackProfiles(iv InputVideo) ([]EncodedProfile, error) {
 		profiles = []EncodedProfile{lowBitrateProfile(video)}
 	}
 	profiles = append(profiles, EncodedProfile{
-		Name:    strconv.FormatInt(video.Height, 10) + "p0",
+		Name:    strconv.FormatInt(nearestEven(video.Height), 10) + "p0",
 		Bitrate: video.Bitrate,
 		FPS:     0,
-		Width:   video.Width,
-		Height:  video.Height,
+		Width:   nearestEven(video.Width),
+		Height:  nearestEven(video.Height),
```

Comment:
Does rounding cause any issues with the Mist/Livepeer pipeline?

Reply:
I think we only pass one of width / height in there and it scales on the transcoder so I don't think this should be an issue

Reply:
oh yeah kinda forgot that, we only pass height to MC too